packages/web-components/fast-components/src/radio/radio.template.ts

```diff
@@ -0,0 +1,29 @@
+import { html, when } from "@microsoft/fast-element";
+import { Radio } from "./radio";
+
+export const RadioTemplate = html<Radio>`
+    <template
+        role="radio"
+        aria-checked="${x => x.checked}"
```

Comment:
These should all be prefixed, except `aria-checked` with `?` so they don't show in the DOM  when they are false.
![image](https://user-images.githubusercontent.com/37851214/80128739-f2743e80-854a-11ea-9441-87f989f1caeb.png)

Reply:
Oh boy that's a new one for me, hadn't seen that anywhere, but found it in button, will do